src/errorpage.cc

```diff
@@ -179,21 +178,23 @@ error_hard_text[] = {
     {
         ERR_REQUEST_START_TIMEOUT,
         "request start timedout"
-    },
+    }
+}};
+
+/// \ingroup ErrorPageInternal
+/// default messages for missing error pages
```

Comment:
```suggestion
/// error messages that may be configured/customized by the admin,
/// like the regular ones, but that also have a hard-coded last-resort default
```

... which raises a question: Why do we add this array instead of adding a `MGR_INDEX` file with the template? What is the difference with the other/regular error templates? The description should probably be adjusted further to reflect/explain this difference.

Reply:
Adjusted.

MGR_INDEX was added at b073fc4, but unfortunately it lacks explanation why we 'do not distribute any preset template'. There is an 'example' script on a wiki [page|https://wiki.squid-cache.org/Features/CacheManager/Index], but it is still unclear why Squid could not be supplied by this script (or its analog).

Reply:
AFAICT, commit b073fc4bde37d3b0dd2f820f90c801d7dada1338 explains that the template is not distributed because "This allows manager apps to provide their own static template with linked scripts and objects." I do not really understand what that means -- why providing a template would prevent some apps from providing their own. I suggest asking that question during the official review.